docs/guide/tensorboard.rst

```diff
@@ -146,6 +147,7 @@ Here is an example of how to store a plot in TensorBoard at regular intervals:
             figure.add_subplot().plot(np.random.random(3))
             # Close the figure after logging it
             self.logger.record("trajectory/figure", Figure(figure, close=True), exclude=("stdout", "log", "json", "csv"))
+            plt.close()
```

Comment:
You shouldn't need plt.close here, if we already told tensorflow to close the figure (close=True)

Reply:
I added it because matplotlib was complaining, even with `close=True`